test/mailers/user_mailer_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
Lets not include the test files if we are not writing test cases.

Reply:
Okay, will take care of this